coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/parsetree/ParseTreeBuilder.java

```diff
@@ -451,7 +453,22 @@ protected SqlNode visitNumber(ASTNode node, ParseContext ctx) {
 
   @Override
   protected SqlNode visitAllColRef(ASTNode node, ParseContext ctx) {
-    return SqlIdentifier.star(ZERO);
+    List<SqlNode> children = visitChildren(node, ctx);
+    // This is to allow t.* and t.col.*
```

Comment:
My Hive 3.1 instance doesn't allow `t.col.*`. It also doesn't allow `schema.table.*` references.

`t.col.*` might be tricky, we should have more end-to-end testing coverage.

Reply:
Good point.  Removed the incorrect comment.